### ISSUE TITLE
B10 Review

### ISSUE THREAD

**Issue by jshreyans551** [jshreyans551/frontend-101#17]
I hadn't seen the additional instructions page in the assignment folder. Saw this task and thought it was interesting so I did it first. Hope there's no bug!

**Comment by sarthak-sehgal** [comment]
Looks great!
I think you're yet to try out the other methods though.

You also might want to add a "Not found" message for the non existent usernames.

**Comment by jshreyans551** [comment]
Yes I've been working on them and will try to add both by tomorrow. And I'll also work on the 'Not Found' message.
Thanks a lot!
